docs/usage/shoot-operations/shoot_updates.md

```diff
@@ -222,6 +220,17 @@ The `ManualInPlaceWorkersUpdated` [constraint](../shoot/shoot_status.md#constrai
 
 The `inPlaceUpdates.pendingWorkerUpdates.manualInPlaceUpdate` field in the `Shoot` status lists the names of worker pools that are pending updates with this strategy.
 
+#### Limitations when Configuring In-Place Updates via Gardener Dashboard
+
+As of now, the Gardener Dashboard's `Worker` panel does not provide an option to set the update strategy or automatically select it based on the chosen machine image. Historically, only the `AutoRollingUpdate` strategy was supported and set as the default for all worker pools. Future improvements, such as the introduction of [machine-image-capabilities](../../proposals/33-machine-image-capabilities.md), are expected to simplify update strategy configuration. Until then, consider the following when configuring in-place updates:
+
+- ##### Configuring In-Place Update for a New Shoot
+  When creating a new shoot, select a `Machine Image` that supports in-place updates in the `Worker` panel and set the desired min/max/surge values. Before saving, use the YAML editor to specify the `updateStrategy` as either `AutoInPlaceUpdate` or `ManualInPlaceUpdate`, following the recommendations outlined in the sections above.
```

Comment:
>  When creating a new shoot, select a `Machine Image` that supports in-place updates in the `Worker` panel and set the desired min/max/surge values. Before saving, use the YAML editor to specify the `updateStrategy` as either `AutoInPlaceUpdate` or `ManualInPlaceUpdate`,

This is literally hand-holding. The User should check the spec and set these values. I am not aware of any docs in Gardener that explain users to save YAML and all.